api/machine/machine.proto

```diff
@@ -791,11 +791,17 @@ message EtcdForfeitLeadershipResponse {
 message EtcdMemberListRequest {
   bool query_local = 1;
 }
-message EtcdMemberList {
+message EtcdMember {
```

Comment:
it might make sense to add some documentation here as we update the API to make our docs more readable